### PR TITLE
add support for aggregated stats

### PR DESCRIPTION
**Pull request by wlan0**
@cjellick

### CODE REVIEW

**Comment by wlan0** [comment]
https://github.com/rancher/rancher/issues/1265

**Comment by wlan0** [comment]
Also fixes https://github.com/rancher/rancher/issues/1527

**Comment by cjellick** [comment]
@wlan0 Can you change the file names to use underscores instead of camel case?

File: stats/stats.go [hunk 1]
```diff
@@ -110,52 +100,27 @@ func (s *StatsHandler) Handle(key string, initialMessage string, incomingMessage
...
-		return "", err
-	}
-
-	if useSystemd() {
-		return fmt.Sprintf("system.slice/docker-%s.scope", container.ID), nil
-	} else {
-		return fmt.Sprintf("docker/%s", container.ID), nil
-	}
-}
+		//because error was not being returned at this point, the for loop using this function never exits.
```
Comment (cjellick) [inline stats/stats.go]: Can you either remove or reword this comment? It's a good explanation of the bug you fixed, but I don't think it belongs in here long term. It won't make sense to someone looking at the code wwho doesnt have the context of the bug.

**Comment by cjellick** [comment]
LGTM

**Pull request merged**